modules/SofaConstraint/GenericConstraintSolver.h

```diff
@@ -130,6 +130,8 @@ class SOFA_CONSTRAINT_API GenericConstraintSolver : public ConstraintSolverImpl
 	Data<int> currentIterations; ///< OUTPUT: current number of constraint groups
 	Data<double> currentError; ///< OUTPUT: current error
     Data<bool> reverseAccumulateOrder; ///< True to accumulate constraints from nodes in reversed order (can be necessary when using multi-mappings or interaction constraints not following the node hierarchy)
+    Data<helper::vector< double >> constraintForces;
```

Comment:
Please use `d_` convention when defining new Data.
See [N5] in https://github.com/sofa-framework/sofa/blob/master/GUIDELINES.md#naming